### DISCUSSION TITLE
Shortest distance - routing.xml

### DISCUSSION THREAD

**Discussion by ghost** [osmandapp/OsmAnd#15033] [Q&A]
Is there a way to create a routing.xml file that will take shortest distance only?

Right now I have this:
```
<?xml version="1.0" encoding="utf-8" ?>
<osmand_routing_config>
  <routingProfile name="TWIKE 3">
  </routingProfile>
</osmand_routing_config>
```
but it doesn't do the trick, so I guess there is some deeper code in OsmAnd that takes over?

Also part of this discussion: https://github.com/osmandapp/OsmAnd/discussions/12577

**Comment by vshcherb** [comment]
Pedestrian profile will does it , so you can copy section of pedestrian profile and modify it

**Reply by ghost** [reply]
I tried but it didn't work either. Result was almost the same as the 'empty' xml file or in some cases even worse (longer). I guess OsmAnd isn't really good for this, at least not in a simple way.
